cfme/common/vm.py

```diff
@@ -315,7 +315,7 @@ def find_quadicon(
             else:
                 raise TemplateNotFound("Template '{}' not found in UI!".format(self.name))
 
-        paginator.results_per_page(1000)
+        paginator.results_per_page('1000 items')
```

Comment:
I'm guilty if it requires item/items. anyway, it has to be fixed in paginator not here.